.github/actions/build-push-image/action.yaml

```diff
@@ -49,18 +49,23 @@ inputs:
   tags:
     description: A list of tags applied to the image
     required: true
+  devbox-build:
```

Comment:
Do we still need this parameter? I understand it was good for testing and evaluating the tool but IMO, now it only adds complexity to the action. I'd say we can remove it and stick to devbox only

Reply:
I agree, we can remove. I can handle it.